README.md

```diff
@@ -209,6 +209,7 @@ Please contact sales@flexera.com to learn more.
 - [Azure VMs Not Using Managed Disks](./operational/azure/vms_without_managed_disks/)
 - [Azure Migrate Integration](./operational/azure/azure_migrate)
 - [AzureAD Group Sync](./operational/azure/azuread_group_sync/)
+- [Sync Azure Tags with Optima](./operational/azure/sync_tags_with_optima/)
```

Comment:
We like to have the provider name in front of the name.  How about renaming to Azure Tag Sync with Optima